src/road/traffic.py

```diff
@@ -54,7 +54,7 @@ class Vehicle():
     def __init__(self, id, node: RoadSegmentNode):
         # Attributes
         self.id = id
-        self.speed = 1 * tw  # WARNING: could have undesirable behavior
+        self.vehicle_speed = 1 # Vehicle speed irrespective of the tile size
```

Comment:
The reason I left the prior comment is resizing the TileGrid could produce unwanted/unexpected results to vehicle movement. Is a strange coupling. I left it there for now with a warning to be changed when needed.